packages/flutter_tools/lib/src/macos/macos_device.dart

```diff
@@ -20,19 +27,13 @@ class MacOSDevice extends Device {
   DeviceLogReader getLogReader({ApplicationPackage app}) => NoOpDeviceLogReader('macos');
 
   @override
-  Future<bool> installApp(ApplicationPackage app) {
-    throw UnimplementedError();
-  }
+  Future<bool> installApp(ApplicationPackage app) async => true;
```

Comment:
Some comments about why these are hard-coded to true could be useful. Is there actually an implementation step that could be needed in the future? If so, then adding a TODO here as well could be a good pointer.

Reply:
Still working on this

Reply:
Done